packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -530,6 +562,45 @@ export function SelectPanel({
             >
               {footer}
             </Box>
+          ) : (
+            <Box
```

Comment:
should we only show this when the variant is multi? feels weird to have this save button here when the selectpanel closes on selection anyways 🤔 
CC: @emilybrick 

https://github.com/user-attachments/assets/89689204-5d08-442e-afed-5a2674f40be8

Reply:
Yesss, good point. I'll hide the footer here but keep the close button in the header 👍

Reply:
Fixed!